src/site/content/en/blog/sms-receiver-announce/index.md

```diff
@@ -0,0 +1,401 @@
+
+```js
+if ('sms' in navigator) {
+  ...
+}
+```
+
+### Process the OTP
+
+The SMS Receiver API itself is simple enough:
+
+```js
+const sms = await navigator.sms.receive();
+```
+
+Once a user taps **Verify**, displayed in the bottom sheet, the promise
+containing the text message will resolve. You can use a regular expression to
+distill the OTP and verify the user. For example, if a text message contains a
+six digit verification code following `otp=`, the code would look like this:
+
+```js
+const code = sms.content.replace(/[\s\S]\*otp=([0-9a-zA-Z]{6})[\s\S]\*$/m, '$1');
+```
+
+You can now submit the code to the server to verify it.
+
+### Designing the SMS message
+
+The API itself should have looked simple enough, but a critical part is to
+format your SMS text message according to a specific convention. The message has
+to be sent after `navigator.sms.receive()` is called and in **the right format**.
+
+The SMS message must be received on the device where `navigator.sms.receive()`
+was called.
+
+The message must adhere to the following formatting:
+
+* The application hash of the user's Chrome instance. (These are static strings.
+  See the table after the example.)
+* The origin part of the URL of the website that invoked the API. It must be
+  preceded by `For: `.
+* The URL must contain a query parameter `otp` whose value is the OTP.
+
+An example message that can be retrieved by the browser would look like this:
+
+```text
+Your OTP is: 123456.
+
+For: https://sms-receiver-demo.glitch.me/?otp=123456&PqEvUq15HeK
+```
+
+The application hash of Chrome instances are static. Use one of these strings
+for development depending on which Chrome build you will be working with.
+
+<table>
+<tr>
+<td markdown="block">
+<strong>Chrome build</strong>
+</td>
+<td markdown="block">
+<strong>APK hash string</strong>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Chrome Canary
+</td>
+<td markdown="block">
+<code>PqEvUq15HeK</code>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Chrome Dev
+</td>
+<td markdown="block">
+
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Chrome Beta
+</td>
+<td markdown="block">
+<code>xFJnfg75+8v</code>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Chrome Stable
+</td>
+<td markdown="block">
+<code>EvsSSj4C6vl</code>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Locally built Chromium
+</td>
+<td markdown="block">
+<code>s3LhKBB0M33</code>
+</td>
+</tr>
+</table>
+
+### Demos
+
+Try various messages with the demo:
+[https://sms-receiver-demo.glitch.me](https://sms-receiver-demo.glitch.me)
+
+You may also fork it and create your version:
+[https://glitch.com/edit/\#!/sms-receiver-demo](https://glitch.com/edit/#!/sms-receiver-demo).
+
+### Enabling support during the origin trial
+
+Starting in Chrome 78, the SMS Receiver API will be available as an origin trial
+on Chrome for Android. Origin trials allow you to try new features and give
+feedback on their usability, practicality, and effectiveness, both to us, and to
```

Comment:
Who is `us` here? I think you mean Chrome, but keep in mind that web.dev is an open web resource, so we need to clarify.